src/Components/__tests__/ToolTipModal.spec.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {mount} from 'enzyme';
+import Modal from '@mui/material/Modal';
+import Backdrop from '@mui/material/Backdrop';
+import Button  from '@mui/material/Button';
+
+import ToolTipModal from '../ToolTipModal';
+
+describe("<ToolTipModal />", () => {
+    let wrapper;
+    beforeEach(() => {
+        wrapper = mount(<ToolTipModal text={"test"} />)
+    })
+
+    it("renders a modal", () => {
+        expect(wrapper.find(Modal)).toHaveLength(1);
```

Comment:
If we're just testing if there is 1 node (i.e. if something exists or not) how do you feel about using the[ `exists` selector? ](https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/exists.html)

Reply:
That selector makes a lot more sense for those type of tests.